src/Caching/SqlServer/src/MonoSqlParameterCollectionExtensions.cs

```diff
@@ -22,23 +22,22 @@ internal static class MonoSqlParameterCollectionExtensions
             this SqlParameterCollection parameters,
             DateTimeOffset utcTime)
         {
-            return parameters.AddWithValue(Columns.Names.ExpiresAtTime, SqlDbType.DateTime, utcTime.UtcDateTime);
+            return parameters.AddWithValue(Columns.Names.ExpiresAtTime, SqlDbType.VarChar, utcTime.UtcDateTime.ToString());
```

Comment:
If the stored date is going to parsed later then this not good to store the date using `DateTime.ToString()` as it will depend on the current culture and parsing can fail later. It is better either to use InvariantCulture when formatting it `utcTime.UtcDateTime.ToString(CultureInfo.InvariantCulture)` or just format the date using "o" or "r" and when parsing using the same specifiers `utcTime.UtcDateTime.ToString("o")`.